src/core/lib/channel/channelz.cc

```diff
@@ -385,52 +385,63 @@ grpc_json* SocketNode::RenderJson() {
   json = data;
   json_iterator = nullptr;
   gpr_timespec ts;
-  if (streams_started_ != 0) {
+  if (gpr_atm_no_barrier_load(&streams_started_) != 0) {
     json_iterator = grpc_json_add_number_string_child(
-        json, json_iterator, "streamsStarted", streams_started_);
-    if (last_local_stream_created_millis_ != 0) {
-      ts = grpc_millis_to_timespec(last_local_stream_created_millis_,
-                                   GPR_CLOCK_REALTIME);
+        json, json_iterator, "streamsStarted",
+        gpr_atm_no_barrier_load(&streams_started_));
```

Comment:
Can we try to do each of these atomic loads only once instead of twice?  For example:

```
int64_t streams_started = gpr_atm_no_barrier_load(&streams_started_);
if (streams_started != 0) {
  json_iterator = grpc_json_add_number_string_child(
      json, json_iterator, "streamsStarted", streams_started);
  // ...
}
```

Reply:
Yeah, I thought about that, but thought that it wasn't worth the effort since it's not a performance-critical path. NBD, I can change that.